Competition2017/src/competition/subsystems/drive/commands/DriveToPositionCommand.java

```diff
@@ -0,0 +1,48 @@
+package competition.subsystems.drive.commands;
+
+import com.google.inject.Inject;
+
+import xbot.common.command.BaseCommand;
+import xbot.common.math.PIDManager;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+import competition.subsystems.drive.DriveSubsystem;
+
+public class DriveToPositionCommand extends BaseCommand{
```

Comment:
This feels more like a "drive for distance" command than a "drive to position" command. A _position_ implies to me that it's a point on the field in both dimensions.